e2e_test/source/nexmark_endless/nexmark_endless_part1.slt

```diff
@@ -2,7 +2,7 @@ include ../../nexmark/create_sources.slt.part
 
 include ../../streaming/nexmark/views/q3.slt.part
 
-sleep 10s
+sleep 15s
```

Comment:
Does it become more unstable? 👀

Reply:
yes, but I forgot to try this: https://github.com/risingwavelabs/risingwave/pull/7124/files#r1058871481

Reply:
no effect, probably because using debug build, the source throughput is limited by the CPU. 
So both increasing `split.num` and reducing `nexmark.min.event.gap.in.ns` has minimum impact.